admin/assets/stylesheets/elements/_printing.scss

```diff
@@ -32,14 +32,23 @@
       font-size: medium;
     }
   }
-  td, tr {
-    min-height: 180px;
-    min-width: 200px;
-    max-height: 180px;
-    max-width: 200px;
-  }
   .pupil-content {
+    text-align: left;
     line-height: 2em;
     font-size: .8em;
   }
+  .qr {
+    width: 5em;
```

Comment:
@GeorgeChatzigiannis Pixels remain the preferred unit to use as it makes it easier to keep our styles consistent with the ones provided by GOV.UK, also easier to use $gutter variables and keep a standardise use of the space.

Reply:
@pris54 i see we are using both em and px everywhere?
I am happy to change this to px but we should also take notice then and update the whole css codebase with px units.
I'll introduce a similar variable to gutter